secedgar/filings/filing_types.py

```diff
@@ -6,145 +6,1016 @@ class FilingType(Enum):
+    FILING_F7A = 'F-7/A' # Registration statement for securities of certain Canadian issuers offered for cash upon the exercise of rights granted to existing security holders under the Securities Act of 1933
+    FILING_F8POS = 'F-8 POS' # Post-effective amendment to a F-8 registration
+    FILING_F8 = 'F-8' # Registration statement for securities of certain Canadian issuers to be issued in exchange offers or a business combination under the Securities Act of 1933
+    FILING_F8A = 'F-8/A' # Registration statement for securities of certain Canadian issuers to be issued in exchange offers or a business combination under the Securities Act of 1933
+    FILING_F80 = 'F-80' # Registration of securities of certain Canadian issuers to be issued in exchange offers or a business combination under the Securities Act of 1933
+    FILING_F80A = 'F-80/A' # Registration of securities of certain Canadian issuers to be issued in exchange offers or a business combination under the Securities Act of 1933
+    FILING_F80POS = 'F-80POS' # Post-effective amendment to a F-80 registration
+    FILING_FN = 'F-N' # Notification of the appointment of an agent for service by certain foreign institutions
+    FILING_FNA = 'F-N/A' # Notification of the appointment of an agent for service by certain foreign institutions
+    FILING_FWP = 'FWP' # Filing under Securities Act Rules 163/433 of free writing prospectuses
+    FILING_FX = 'F-X' # For appointment of agent for service of process by issuers registering securities
+    FILING_FXA = 'F-X/A' # For appointment of agent for service of process by issuers registering securities
+    FILING_IRANNOTICE = 'IRANNOTICE' # Notice of disclosure filed in Exchange Act quarterly and annual reports pursuant to Section 219 of the Iran Threat Reduction and Syria Human Rights Act of 2012 and Section 13(r) of the Exchange Act
+    FILING_MA = 'MA' # Municipal Advisor registration required pursuant to Section 15B(a)(2) of the Securities Exchange Act of 1934 and the Rules Thereunder. Application for Municipal Advisor Registration (MA), Annual Update of Municipal Advisor Registration (MA-A), and Amendment of a Prior Application for Registration (MA/A)
+    FILING_MAA = 'MA-A' # Municipal Advisor registration required pursuant to Section 15B(a)(2) of the Securities Exchange Act of 1934 and the Rules Thereunder. Application for Municipal Advisor Registration (MA), Annual Update of Municipal Advisor Registration (MA-A), and Amendment of a Prior Application for Registration (MA/A)
+    FILING_MAA = 'MA/A' # Municipal Advisor registration required pursuant to Section 15B(a)(2) of the Securities Exchange Act of 1934 and the Rules Thereunder. Application for Municipal Advisor Registration (MA), Annual Update of Municipal Advisor Registration (MA-A), and Amendment of a Prior Application for Registration (MA/A)
```

Comment:
The script is creating a few duplicates like this. I think we should create a script which we include in a scripts module that will generate this file for us. However, we will need to fix this duplicates issue.

Also, what is the URL you are getting the Excel/CSV file from? Would be better to just send a request to get the data rather than making everybody store the file locally when they pip install this package.

Reply:
I used a tool, `tabula`, to extract the CSV file from the EDGAR official documentation.

![image](https://user-images.githubusercontent.com/13869303/111392701-87b18180-868d-11eb-882f-5cad72a4bc56.png)